packages/react-magma-dom/src/components/Typography/index.tsx

```diff
@@ -123,7 +125,13 @@ const baseParagraphStyles = props => css`
 export const paragraphLargeStyles = props => css`
   ${baseParagraphStyles(props)}
 
-  margin: ${props.noMargins ? '0' : `${props.theme.spaceScale.spacing06} 0`};
+  margin: ${props.noMargins
```

Comment:
I think for the logic for all these, we should update it to allow adopters to use both `noBottomMargin` and `noTopMargin`. I know it seems counterintuitive because there's already a `noMargins` prop, but otherwise it's unclear to someone using both which one takes precedence.